### DISCUSSION TITLE
Would be nice to be able to filter out forked repos when searching with "org:"

### DISCUSSION THREAD

**Discussion by doegox** [community/community#10868] [Other Feature Feedback, Questions, & Ideas] [answered]
Some orgs/users have many forked repos, it would be nice to have a way to limit the scope of a search within their repos to the ones really created by them and not those forks. Something to be used in combination with `org:`

**Comment by look** [comment]
This is on our roadmap. I expect the syntax to be something like `is:fork` so you would be able to do a search like `search terms org:myorg NOT is:fork`. Would that work for you?

**Reply by bgianfo** [reply]
`is:fork` is exactly what I was just looking for, eagerly awaiting such a feature!

**Comment by doegox** [comment]
great news, yes !

**Comment by mgol** [comment]
This is possible in the old search and is definitely a blocker for me to use the new Code Search for a lot of queries that I do. `is:fork` would work!

**Comment by look** [comment, marked as answer]
Thanks for your patience, everyone. There is now an option to exclude forks (or limit your search to forks).

To exclude forks from your results:

```
NOT is:fork
```

To search only fork repositories:

```
is:fork
```

For more information, please [see the documentation](https://docs.github.com/en/search-github/github-code-search/understanding-github-code-search-syntax#is-qualifier).

**Reply by ackvf** [reply]
Btw, it seems that `fork:true`, `fork:false`, `fork:only` are the correct options now.
